boards/arm/nrf5340dk_nrf5340/nrf5340_cpunet_reset.c

```diff
@@ -28,15 +22,6 @@ LOG_MODULE_REGISTER(nrf5340dk_nrf5340_cpuapp, CONFIG_LOG_DEFAULT_LEVEL);
 static void remoteproc_mgr_config(void)
 {
 #if !defined(CONFIG_TRUSTED_EXECUTION_NONSECURE) || defined(CONFIG_BUILD_WITH_TFM)
-	/* UARTE */
-	/* Assign specific GPIOs that will be used to get UARTE from
-	 * nRF5340 Network MCU.
-	 */
-	soc_secure_gpio_pin_mcu_select(CPUNET_UARTE_PIN_TX, NRF_GPIO_PIN_MCUSEL_NETWORK);
-	soc_secure_gpio_pin_mcu_select(CPUNET_UARTE_PIN_RX, NRF_GPIO_PIN_MCUSEL_NETWORK);
-	soc_secure_gpio_pin_mcu_select(CPUNET_UARTE_PIN_RTS, NRF_GPIO_PIN_MCUSEL_NETWORK);
-	soc_secure_gpio_pin_mcu_select(CPUNET_UARTE_PIN_CTS, NRF_GPIO_PIN_MCUSEL_NETWORK);
```

Comment:
This patch is number 2 in the series, but the calls to soc_secure_gpio_pin_mcu_select() are not added to soc.c until patch number 4. There is therefore a bisectability problem with this PR, but it seems minor and not worth blocking over.

Reply:
I fixed the bisectability problem, the cleaned up commits are present at my fork: https://github.com/jciupis/zephyr/tree/pr/nordic-nrf-gpio-forwarder 

However, apparently there's a bug at Github that might cause a PR to not update when new commits are pushed: https://github.community/t/pull-request-isnt-updating-with-new-commits/226239, and I'm experiencing it now. The latest changes are now shown at all in the PR, commits SHA are not updated, CI is not run, pushes are not reported. I'll wait a couple of hours to see if it's just a temporary glitch, but opening a new PR to replace this one might be necessary.

Reply:
@jciupis Could you link to the new PR?

Reply:
@joerchan https://github.com/zephyrproject-rtos/zephyr/pull/43047